src/components/manager/projects/Project.js

```diff
@@ -141,9 +141,7 @@ class Project extends Component {
                                         project={this.props.value}
                                         value={{
```

Comment:
needless spread now... unless we need to copy the object for some reason?

Reply:
The spread copy is not required to my knowledge, removed it